code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/ApiAuthenticator.java

```diff
@@ -145,7 +147,7 @@ protected Account getAccount(ApiRequest request) {
             }
         }
 
-        return account;
+        return null;
```

Comment:
The previous logic always returned account here which was always null. This is more clear that that is the case.